sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -347,6 +323,43 @@ namespace Azure { namespace Data { namespace Tables {
       std::string m_value;
     };
 
+    /**
+     * @brief Table Entity Data Type.
+     */
+    class TableEntityDataType final
+        : public Azure::Core::_internal::ExtendableEnumeration<TableEntityDataType> {
+    public:
+      /**
+       * @brief Construct a new TableEntityDataType object
+       */
+      TableEntityDataType() = default;
+      /**
+       * @brief Construct a new TableEntityDataType object
+       *
+       * @param tableEntityDataType entity data type string.
+       */
+      explicit TableEntityDataType(std::string tableEntityDataType)
+          : ExtendableEnumeration(std::move(tableEntityDataType))
+      {
+      }
+      /** Constant value of type TableEntityDataType:EdmBinary */
+      AZ_DATA_TABLES_DLLEXPORT const static TableEntityDataType EdmBinary;
```

Comment:
Should we just name them "Binary", "Boolean", "DateTime", i.e. without the "Edm" prefix?

Reply:
that is the name in the documentation